README.md

```diff
@@ -27,29 +27,22 @@ Here are the most commonly used components:
 Here are the [instrumentation
 libraries](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/glossary.md#instrumentation-library):
 
-* [OpenTelemetry.Instrumentation.AspNet](./src/OpenTelemetry.Instrumentation.AspNet/README.md)
-  * ASP.NET [:link:](https://docs.microsoft.com/aspnet/overview)
-* [OpenTelemetry.Instrumentation.AspNetCore](./src/OpenTelemetry.Instrumentation.AspNetCore/README.md)
-  * ASP.NET Core [:link:](https://docs.microsoft.com/en-us/aspnet/core)
-* [OpenTelemetry.Instrumentation.Grpc](./src/OpenTelemetry.Instrumentation.Grpc/README.md)
-  * gRPC for .NET [:link:](https://github.com/grpc/grpc-dotnet)
-* [OpenTelemetry.Instrumentation.Http](./src/OpenTelemetry.Instrumentation.Http/README.md)
-  * System.Net.Http.HttpClient [:link:](https://docs.microsoft.com/dotnet/api/system.net.http.httpclient)
-  * System.Net.HttpWebRequest [:link:](https://docs.microsoft.com/dotnet/api/system.net.httpwebrequest)
-* [OpenTelemetry.Instrumentation.SqlClient](./src/OpenTelemetry.Instrumentation.SqlClient/README.md)
-  * Microsoft.Data.SqlClient [:link:](https://www.nuget.org/packages/Microsoft.Data.SqlClient)
-  * System.Data.SqlClient [:link:](https://www.nuget.org/packages/System.Data.SqlClient)
-* [OpenTelemetry.Instrumentation.StackExchangeRedis](./src/OpenTelemetry.Instrumentation.StackExchangeRedis/README.md)
-  * StackExchange.Redis [:link:](https://www.nuget.org/packages/StackExchange.Redis)
+* [ASP.NET](./src/OpenTelemetry.Instrumentation.AspNet/README.md)
+* [ASP.NET Core](./src/OpenTelemetry.Instrumentation.AspNetCore/README.md)
+* [gRPC](./src/OpenTelemetry.Instrumentation.Grpc/README.md)
+* [HTTP client](./src/OpenTelemetry.Instrumentation.Http/README.md)

```

Comment:
Don't know how to say that it's not just http client. We can instrument http web request as well

Reply:
mm. You are right. In main page, i think simply HttpClient is fine. HttpClient is the main level api, and can be used in .net core and .net framework. Underneath, httpwebrequest gets used. This needs to described in the http client readme in a future PR, which should also contain warning about how it can be broken when using custom handler etc.

Another alternate is HttpClient .NET/.NET Framework? or simply HttpClient..
@reyang make `HttpClient` single word?

Reply:
That's a good point. To be honest I don't have strong opinion, maybe we can just call it `HTTP`? Happy to send a small fix if that could bring better clarity.

Reply:
calling it just `HTTP` sounds better alternative, given the complexity. Main page simply says `HTTP`,. We can describe more in the details page.

Reply:
#914